<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="touchButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>121</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>touchBUtton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backlightButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>121</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>backlightButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="beepButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>60</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>beepButton</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
